### PR TITLE
Jenkins migration

### PR DESCRIPTION
**Pull request by CyanDevs**

### CODE REVIEW

**Review by radhikaj — APPROVED** [review]

File: .jenkins/infrastructure/build_azure_managed_images.Jenkinsfile [hunk 1]
```diff
@@ -148,7 +149,7 @@ def buildWindowsManagedImage(String os_series, String img_name_suffix, String la
                                 --resource-group ${VM_RG_NAME} \
                                 --location ${REGION} \
                                 --name ${VM_NAME} \
-                                --size Standard_DC4s_v2 \
+                                --size Standard_DC4s_v3 \
```
Comment (RRathna) [inline .jenkins/infrastructure/build_azure_managed_images.Jenkinsfile:152]: Note: Leaving this here for future reference. I found that scripts/windows-install-prerequisites.ps1 failed sporadically for DC4s_v3. This was caused by not being able to find AESMService.

**Review by RRathna — APPROVED** [review]

File: .jenkins/infrastructure/build_azure_managed_images.Jenkinsfile (hunk 1 above)
Comment (CyanDevs) [reply .jenkins/infrastructure/build_azure_managed_images.Jenkinsfile:151]: Thanks for the heads up Rathna!

**Pull request merged**